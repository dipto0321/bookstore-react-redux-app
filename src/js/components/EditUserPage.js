import React from 'react';
import UserForm from './UserForm';

const EditUserPage = () => (
  <div>
    <h2>Edit Account</h2>
    <UserForm />
  </div>
);

export default EditUserPage;
