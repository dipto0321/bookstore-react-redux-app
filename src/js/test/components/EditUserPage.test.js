import React from 'react';
import { shallow } from 'enzyme';
import EditUserPage from '../../components/EditUserPage';

describe('EditUserPage component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<EditUserPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
