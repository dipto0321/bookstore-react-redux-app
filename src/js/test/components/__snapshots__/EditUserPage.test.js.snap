// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditUserPage component should render correctly 1`] = `
<div>
  <h2>
    Edit Account
  </h2>
  <ConnectFunction />
</div>
`;
